lib/Runtime/PlatformAgnostic/RuntimePlatformAgnosticPch.h

```diff
@@ -15,6 +17,42 @@
 #pragma warning(pop)
 #endif
 
+// Minimal definitions to use AssertMsg in the PAIL
+#ifndef _WIN32
+#define DbgRaiseAssertionFailure() __builtin_trap()
+#define __analysis_assume(x)
+#define __in
+#define __inout
+
+#ifndef TRUE
```

Comment:
Actually, no, not going to do that since I don't want a dependency on an ICU header in the pch (we want ICU to be optional). I'd say leave this as is and we can remove as part of cleaning up the Assertions code later.